PROJECT

```diff
@@ -54,4 +51,12 @@ resources:
   kind: VerticaDB
   path: github.com/vertica/vertica-kubernetes/api/v1
   version: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: vertica.com
+  kind: VerticaRestorePointsQuery
+  path: github.com/vertica/vertica-kubernetes/api/v1
```

Comment:
Lets use v1beta1 as the initial version. We'll graduate to v1 in a subsequent release.